Xamarin.Forms.Platform.Android/ResourceManager.cs

```diff
@@ -89,9 +90,17 @@ public static Drawable GetDrawable(this Resources resource, string name)
 			return AndroidAppCompat.GetDrawable(Forms.Context, id);
 		}
 
+		public static void LogInfo(string message)
```

Comment:
```suggestion
		[EditorBrowsable(EditorBrowsableState.Never)]
		public static void LogInfo(string message)
```

Reply:
oops :-/

If no other functional changes are requested we'll make this change before merging so the build/uitests can finish